Universal/usbw-methods/prebuilt-manual.md

```diff
@@ -0,0 +1,31 @@
+# Fixing USB Wake: Prebuilt and Manual
+
+* [Prebuilt](#prebuilt)
+* [Manual](#manual)
+  * [Grabbing the sample SSDT](#grabbing-the-sample-ssdt)
+  * [Compiling the SSDT](#compiling-the-ssdt)
+* [Wrapping up](#wrapping-up)
+
+## Prebuilt
+
+By far the easiest way to get SSDT-USBW is just downloading the below file:
+
+* [SSDT-USBW.aml](https://github.com/dortania/Getting-Started-With-ACPI/blob/master/extra-files/compiled/SSDT-USBW.aml)
+
+This prebuilt file is just a precompiled version of [SSDT-USBW](https://raw.githubusercontent.com/osy/USBWakeFixup/master/SSDT-USBW.dsl).
```

Comment:
I think just providing the prebuilt method here is fine, since the decompiled version is linked here.